webpack.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require("path");
+
+module.exports = {
+  entry: "./ext/src/content/content",
+  mode: "development",
```

Comment:
Would it allow for more flexibility if we don’t set the `mode` here? Will `npm run build` be running in `development` mode?

Reply:
If you don't set `mode` at all, it defaults to `production` which causes Chrome to error when installing the extension because of a non-UTF8 file encoding. I've tried several webpack plugins that force the encoding on the build and they don't seem to work. I'm not sure how to fix it, but we might be able to get away with just some of the performance optimizations by using `mode: "none",` and manually enabling settings that don't cause issues with Chrome.